src/coreComponents/fileIO/vtk/VTKVTMWriter.hpp

```diff
@@ -33,7 +33,7 @@ class VTKVTMWriter
    * @brief Build the VTM Writer
    * @param[in] filePath path to the file
    */
-  VTKVTMWriter( string const & filePath );
+  explicit VTKVTMWriter( string const & filePath );
```

Comment:
Why didn't you change `string const & filePath` to `string filePath` like you did for others?

Reply:
Indeed, I should change. I tend to pass-by-value into functions that make a direct copy of the argument, which is also the case for this constructor. I didn't notice because I didn't modify anything in `VTKVTMWriter.cpp` file, which would cause me to spot this pattern.